### DISCUSSION TITLE
Please allow more customized email notifications

### DISCUSSION THREAD

**Discussion by cwolfenberger** [dailydotdev/daily#1214] [Feature Requests]
It would be beneficial to those of us with busy inboxes to have the ability to further customize (drill down) on email notifications by type.
I would like to be able to deactivate email notifications for Likes/Upvotes while still receiving comment replies/mentions, rather than a catch-all bucket of "Activity (mentions, replies, upvotes, etc.)".

**Comment by idoshamun** [comment]
Hi @cwolfenberger,

Thanks a lot for the suggestion. We would consider it 🙏
